pysqa/utils/remote.py

```diff
@@ -158,6 +158,10 @@ def _transfer_files(self, file_dict, sftp=None, transfer_back=False):
             else:
                 ssh = self._open_ssh_connection()
             sftp_client = ssh.open_sftp()
+            sftp_client.sock.in_window_size = 8388608
+            sftp_client.sock.out_window_size = 8388608
+            sftp_client.sock.in_max_packet_size = 131072
+            sftp_client.sock.out_max_packet_size = 131072
```

Comment:
I do not like hard coding these values. Can we add them in the `queue.yaml` file?